src/App.jsx

```diff
@@ -27,12 +27,12 @@ function ApolloWrapper() {
   );
 }
 
-function ErrorBoundry() {
+function ErrorBoundaryWrapper() {
   return (
     <ErrorBoundary>
```

Comment:
Let's make sure that `<ErrorBoundary />` isn't suppressing errors. It should be using the logger to log all errors (in development). In production, it should log all errors to an external service (or be configured to log them to an external service... we don't have that service wired up yet :) )